Lib/test/test_dis.py

```diff
@@ -331,6 +333,12 @@ def _fstring(a, b, c, d):
 def _g(x):
     yield x
 
+async def _ag(x):
+    yield x
+
+async def _co(x):
+    await _ag(x)
```

Comment:
Looking more closely, this should really be:

    async def _co(x):
        async for item in _ag(x):
            pass

The current code will happily compile (and hence disassemble), but emits `TypeError: object async_generator can't be used in 'await' expression` if you actually try to run it.